src/components/UploadSection.svelte

```diff
@@ -1,10 +1,19 @@
 <script lang="ts">
     import { onMount } from "svelte";
-    import { getUploadRequest } from "../lib/server/ApiCLient";
-    import { type UploadRequest } from "../lib/server/ApiCLient";
+    import {
+        getUploadRequest,
+        postInsertRequest,
+        pushMediaFilesToS3,
+        type PostResponse,
+        type PostRequest,
+        type UploadRequest,
+    } from "../lib/server/ApiCLient";
+    import exifr from "exifr";
 
     let files = [];
     let uploadedFiles: UploadRequest[];
+    let postRequest: Array<PostRequest> = [];
+    let postResponse: PostResponse[];
```

Comment:
There is no point in making these fields global. Please move them to the function where they are used.